tests/upload_test.go

```diff
@@ -60,4 +61,45 @@ var _ = Describe("Upload tests", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(deleted).To(BeTrue())
 	})
+
+	It("Upload to PVC should fail", func() {
```

Comment:
Slight tweak to "success" test.  Maybe table driven makes sense?

Reply:
done